xaitk_saliency/interfaces/gen_descriptor_sim_sal.py

```diff
@@ -10,35 +10,36 @@ class GenerateDescriptorSimilaritySaliency (Plugfigurable):
     black-box feature-vectors from multiple references and perturbations into
     saliency heatmaps.
 
-    This transformation requires two reference images, translated into
-    feature-vectors via some black-box means, between which we are trying to
-    discern the feature-space saliency.
-    This also requires the feature-vectors for perturbed images as well as the
-    masks of the perturbations as would be output from a
-    :class:`xaitk_saliency.interfaces.perturb_image.PerturbImage`
+    This transformation requires a reference image, and a number of query images
+    all translated into feature-vectors via some black-box means.
```

Comment:
I noticed that we are sometimes inconsistent in whether we use "feature vector(s)" or "feature-vector(s)". I think all instances prior to this instance use the un-hyphenated version.